configure.ac

```diff
@@ -14,17 +14,17 @@ if test "x$DOTNET" = "xno"; then
 fi
 
 # Check dotnet version
-DOTNET_MINIMUM_VERSION=6.0
+DOTNET_MINIMUM_VERSION=8.0
 DOTNET_VERSION=$($DOTNET --version)
 AX_COMPARE_VERSION([$DOTNET_VERSION], [ge], [$DOTNET_MINIMUM_VERSION],
     AC_MSG_NOTICE([Found dotnet $DOTNET_VERSION]),
     AC_MSG_ERROR([dotnet $DOTNET_MINIMUM_VERSION or higher is required (found version $DOTNET_VERSION).])
 )
 
 # Select target framework based on the dotnet version.
-DOTNET_TARGET_FRAMEWORK=net6.0
-AX_COMPARE_VERSION([$DOTNET_VERSION], [ge], [7.0],
-                   [DOTNET_TARGET_FRAMEWORK=net7.0], [])
+DOTNET_TARGET_FRAMEWORK=net8.0
```

Comment:
Similarly, I think this needs to be `net6.0` since the AX_COMPARE_VERSION switches to net8.0 if the version check succeeds